server/tests/routes.test.js

```diff
@@ -80,6 +81,11 @@ describe("API POST Data", () => {
     });
     expect(res.statusCode).toEqual(CONSTANTS.HTTP_SUCCESS);
     expect(res.text).toEqual("TEST_ORDER");
+    // Be nice and delete this order to avoid side effects
+    const addedOrders = await ordersCollection
```

Comment:
why is this needed?

Reply:
Currently we are doing:
1) Clear DB in yarn pretest script
2) Populate DB in jest globalSetup script
3) Run tests

OK:
`DB.test.js` -> `routes.test.js`
Problem:
`routes.test.js` -> `DB.test.js` as we are adding an item to Orders Collection in one of our tests in `routes.test.js`

Solutions:
1) Avoid side effects by dropping this document after we assert it has been successfully added
2) Refactor code such that we clear DB and re-populate in jest `beforeAll` for each test suite

For now, I've taken the earlier approach

Reply:
actually, interesting point on test ordering, raised as issue in #105, the referenced approach seems feasible, might be a better way than the current fix

Reply:
SGTM! 
2# seems like a better approach in long run! Perhaps just copy over the DB from DB.test.js into a new one in beforeAll and then delete it in AfterAll. The items stay neat then.